osquery/utils/aws_util.cpp

```diff
@@ -219,7 +221,7 @@ OsquerySTSAWSCredentialsProvider::GetAWSCredentials() {
       initAwsSdk();
     }
 
-    makeAWSClient<Aws::STS::STSClient>(client_, false);
+    makeAWSClient<Aws::STS::STSClient>(client_, "", false);
```

Comment:
You don't check the status of the return here, if it fails will `client_` be dereferenced on a `nullptr`?